shopinvader_guest_mode/tests/test_guest_service.py

```diff
@@ -73,3 +73,14 @@ def test_guest_mode_allowed(self):
         self.backend.is_guest_mode_allowed = True
         binding = self._create_guest()
         self.assertTrue(binding.is_guest)
+
+    def test_create_account_after_guest(self):
+        self.service.dispatch("create", params=self.data)["data"]
+        with self.work_on_services(
+            partner=None, shopinvader_session=self.shopinvader_session
+        ) as work:
+            customer_service = work.component(usage="customer")
+        data = self.data.copy()
+        data["is_company"] = False
+        data["external_id"] = "D5CdkqOEL"
+        customer_service.dispatch("create", params=data)
```

Comment:
hmm where's the test? :thinking:

Reply:
Well, I admit it is not very clear but the goal of this test is to validate the non-failing creation of the user account, not really its content.
If the test is not good, this line raises an exception.

Reply:
shouldn't you test that you have an existing record matching the same email which is not archived, then create a new one w/ the same email and make sure the old one is archived?

Reply:
ping @Cedric-Pigeon